src/components/scoresheets/scoresheets.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { fetchScoresheet, fetchScoresheets, removeScoresheet } from '../../actions/scoresheet_actions';
+
+//=========================================
+// props / actions
+
+const mapStateToProps = (state) => {
+  return {
+    scoresheets: state.scoresheets,
+    currentUser: state.auth.currentUser,
+  }
+};
+
+const mapDispatchToProps = (dispatch) => ({
+  fetchScoresheet: (scoresheetId) => dispatch(fetchScoresheet(scoresheetId)),
+  fetchScoresheets: (userId) => dispatch(fetchScoresheets(userId)),
+  removeScoresheet: (scoresheetId) => dispatch(removeScoresheet(scoresheetId)),
+});
+
+//=========================================
+// component
+
+class Scoresheets extends React.Component {
+
+  static propTypes = {
+
```

Comment:
Why is this empty? Are you not sure how to do them? We are definitely expecting props in this component so let's clarify what we are expecting. We can talk about how to do this.